templates/haas_cft.json

```diff
@@ -3,11 +3,19 @@
     "Mappings": {
         "RegionMap": {
             "us-east-1": {
```

Comment:
@oxhead -- does this limit us to this region? Do we need to parameterize the region?

Reply:
This is used as a lookup table (pick the right ami) but not for limiting the region.